torch/onnx/symbolic_opset9.py

```diff
@@ -163,6 +163,10 @@ def sqrt(g, self):
     return g.op("Sqrt", self)
 
 
+def rsqrt(g, self):
+    return div(g, g.op("Constant", value_t=torch.tensor(1.)), sqrt(g, self))
```

Comment:
If type of self is double, is there a possibility of type mismatch between the two arguments of the `div` op given that `Constant` is always of type float?

Reply:
I am creating a constant of the correct dtype now. Thanks!